sagemaker/launch/launch_infer.py

```diff
@@ -20,8 +20,10 @@
 import boto3 # pylint: disable=import-error
 import sagemaker
 from sagemaker.pytorch.estimator import PyTorch
+from sagemaker.local.local_session import LocalSession
```

Comment:
Why do we import the LocalSession here in launch_infer.py instead of in the common parser?

Reply:
LocalSession is not used.